### PR TITLE
Added:filter and sort functionality

### PR DESCRIPTION
**Pull request by talrejah02**
Features Added 
- Filter
- sort
- color

### CODE REVIEW

File: src/utils/filter.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    }
+  };
+  
+  const filterByTag = (list, state) => {
+    if (state.tag === "all") return list;
+    return list.filter((note) => note.tags.includes(state.tag));
+  };
+  
+const filterByPriority = (list, state) => {
+    console.log(state)
```
Comment (RahulKuvlekar) [inline src/utils/filter.js:15]: - Remove the console log

File: src/components/notesEditor/notesEditor.jsx [hunk 2]
```diff
@@ -73,12 +103,13 @@ function Noteseditor() {
...
           toast({
-            position: 'top-right',
+            position: "top-right",
             title: "Note Created",
             status: "success",
             duration: 2000,
             isClosable: true,
-          })
+          });
+          console.log(response.data.notes);
```
Comment (RahulKuvlekar) [inline src/components/notesEditor/notesEditor.jsx:112]: - Remove the console log

**Review by AbhishekkGautam — APPROVED** [review]
code looks clean & well structured. Good to merge

**Review by RahulKuvlekar — COMMENTED** [review]
- When the Note is been Added, Delete the previous note text from Input
- Also Filter Menu is going below the Priority and Tags Inputs
 
Screenshot 2022-05-21 at 3 56 56 PM

**Review by gopalbbharadva — APPROVED** [review]
Code is well structured . Good to go.

**Review by preethasri — APPROVED** [review]
good to merge

**Review by ShreyashMahajan — COMMENTED** [review]
good to merge

**Pull request merged**